about us/about_us.js

```diff
@@ -0,0 +1,341 @@
+      },
+    ],
+  },
+  {
+    obstacleName: "Impatience",
+    obstacleFix:
+      "Fix: sustainable things worth having take time and great deal of effort.",
+  },
+  {
+    obstacleName: "Focus on money",
+    obstacleExplained: "Focus on money rather than the study and the work.",
+    obstacleFix:
+      "Fix: if you just want to make more money, it will be hard to get through all the challenges along the way. If you find a way to enjoy the process, then money will be an inevitable reward at the end.",
+  },
+  {
+    obstacleName: "Amount of work",
+    obstacleExplained: "Not understanding how much work it is before starting.",
+    obstacleFix:
+      "Fix: it will take 6-24 months day and night studying before you become hireable.",
+  },
+  {
+    obstacleName: "Not enough resources",
+    obstacleSubCategories: [
+      {
+        name: "Money/living expenses.",
+        obstacleFix: "Fix: have savings. The program can help.",
+      },
+      {
+        name: "Computer/internet",
+        subCategoryExplained: "Access to a computer and the internet",
+        obstacleFix: "Fix: save and buy. The program can help.",
+      },
+      {
+        name: "Time to study and practice.",
+        obstacleFix:
+          "Fix: lower your overhead. Reduce everything else you do, e.g. your day job hours.",
+      },
+      {
+        name: "Resources",
+        subCategoryExplained: "curriculum, books, tutorials, mentorship.",
+        obstacleFix:
+          "Fix: the internet provides a wide variety of free or cheap resources. The program can help.",
+      },
+      {
+        name: "Lack of english literacy.",
+        obstacleFix: "Fix: learn english.",
+      },
+      {
+        name: "Emotional support",
+        subCategoryExplained: "family, friends, teachers, etc.",
+        obstacleFix:
+          "Fix: surround yourself with supportive people. Stay away from negative ones. This might be difficult, but is worth it.",
+      },
+    ],
+  },
+  {
+    obstacleName: "Industry prejudices",
+    obstacleExplained:
+      "Potential industry prejudices against the candidate’s age and gender.",
+    obstacleFix:
+      "Fix: competent professionals with good work ethics are in high demand. Become so good at what you do, that no one can dismiss you.",
+  },
+
+  {
+    obstacleName: "Poor planning",
+    obstacleExplained: "Planning poorly and not adapting it along the way.",
+    obstacleFix: "Fix: keep a feedback loop and keep improving.",
+  },
+];
+
+const knownObstaclesBox = document.getElementById("known-obstacles");
+const withSubcategoriesBox = document.getElementById("subcategories");
+
+let mutualId = 0;
+let quantity = 0;
+const firstColumn = document.createElement("div");
+firstColumn.classList.add("first-column");
+const secondColumn = document.createElement("div");
+secondColumn.classList.add("second-column");
```

Comment:
These are only being used in the renderKnownObstacles function. Do they need to happen in global scope? 
If yes, why not put them in html? why create them dynamically with js?